### PR TITLE
Fix vulnerabilities

### PR DESCRIPTION
**Pull request by OlfatProgressus**

### CODE REVIEW

File: pr-dhl-woocommerce/assets/css/jquery.fancybox.min.css
Comment (Abdalsalaam) [inline pr-dhl-woocommerce/assets/css/jquery.fancybox.min.css:1]: Is there any reason for adding these files? 

pr-dhl-woocommerce/assets/css/jquery.fancybox.min.css
pr-dhl-woocommerce/assets/img/icon.svg
pr-dhl-woocommerce/assets/js/jquery.fancybox.min.js

Comment (OlfatProgressus) [reply pr-dhl-woocommerce/assets/css/jquery.fancybox.min.css:1]: @Abdalsalaam  PCP plugin error when include external links

File: pr-dhl-woocommerce/includes/class-pr-dhl-wc-order-deutsche-post.php [hunk 1]
```diff
@@ -424,36 +440,36 @@ public function order_items_table( $items, $current_wc_order = null ) {
...
 			<tr>
-				<th><?php _e( 'Item', 'dhl-for-woocommerce' ) ?></th>
-				<th><?php _e( 'Actions', 'dhl-for-woocommerce' ) ?></th>
+				<th><?php esc_html_e( 'Item', 'dhl-for-woocommerce' ); ?></th>
+				<th><?php esc_html_e( 'Actions', 'dhl-for-woocommerce' ); ?></th>
 			</tr>
 			</thead>
 			<tbody>
-			<?php echo implode( '', $table_rows ) ?>
+			<?php echo implode( '', wp_kses_post( $table_rows ) ); ?>
```
Comment (Abdalsalaam) [inline pr-dhl-woocommerce/includes/class-pr-dhl-wc-order-deutsche-post.php:448]: This change caused Fatal error
image

$table_rows is array of HTML table rows , and no need to use **wp_kses_post** here

@OlfatProgressus

File: pr-dhl-woocommerce/includes/abstract-pr-dhl-wc-order.php [hunk 2]
```diff
@@ -15,243 +15,255 @@
...
+
+				$print_button = '<a href="' . $this->get_download_label_url( $order_id ) . '" id="dhl-label-print" class="button button-primary" download target="_blank">' . esc_html__( 'Download Label', 'dhl-for-woocommerce' ) . '</a>';
 			}
 
-			wp_enqueue_script( 'wc-shipment-dhl-label-js', PR_DHL_PLUGIN_DIR_URL . '/assets/js/pr-dhl.js', array('jquery'), PR_DHL_VERSION );
-			wp_localize_script( 'wc-shipment-dhl-label-js', 'dhl_label_data', $dhl_label_data );
+			$dhl_label_data = array(
+				'main_button'  => wp_kses_post( $main_button ),
+				'delete_label' => wp_kses_post( $delete_label ),
+				'print_button' => wp_kses_post( $print_button ),
```
Comment (Abdalsalaam) [inline pr-dhl-woocommerce/includes/abstract-pr-dhl-wc-order.php:162]: these are 3 buttons created within the code, not the user, and the label/text for these buttons are already escaped so there is no need to use **wp_kses_post**

**Review by Abdalsalaam — CHANGES_REQUESTED** [review]

File: pr-dhl-woocommerce/includes/class-pr-dhl-wc-order-deutsche-post.php [hunk 3]
```diff
@@ -303,7 +309,7 @@ public function country_available_packet_return(){
 	 * @since [*next-version*]
 	 */
 	public function dhl_order_meta_box() {
-		echo $this->dhl_order_meta_box_table();
+		echo wp_kses_post( $this->dhl_order_meta_box_table() );
```
Comment (Abdalsalaam) [inline pr-dhl-woocommerce/includes/class-pr-dhl-wc-order-deutsche-post.php:312]:
```suggestion
		echo $this->dhl_order_meta_box_table();
```

We should not sanitize HTML content that is made by the plugin itself
image

**Review by Abdalsalaam — CHANGES_REQUESTED** [review]

**Review by Abdalsalaam — APPROVED** [review]
I pushed some changes and fixes then I tested Paket and Deutsche Post functionalities, all looks good
thank you

**Pull request merged**